pkgs/data/fonts/noto-fonts/default.nix

```diff
@@ -125,4 +125,27 @@ in
       maintainers = with maintainers; [ mathnerd314 ];
     };
   };
+  noto-fonts-emoji-blob = stdenv.mkDerivation rec {
+    pname = "noto-fonts-emoji-blob";
+    version = "2019-06-14-Emoji-12";
+
+    src = fetchurl {
+      url = "https://github.com/C1710/blobmoji/releases/download/v${version}/Blobmoji.ttf";
+      sha256 = "0snvymglmvpnfgsriw2cnnqm0f4llav0jvzir6mpd17mqqhhabbh";
+    };
+
+    dontUnpack = true;
+
+    installPhase = ''
+      mkdir -p $out/share/fonts/blobmoji
+      cp $src $out/share/fonts/blobmoji/Blobmoji.ttf
+    '';
+
+    meta = with stdenv.lib; {
```

Comment:
You could add the github page as homepage here

Reply:
👍